lambda/onMessage.ts

```diff
@@ -95,7 +98,16 @@ const success =
 
 const h = async (event: AuthorizedEvent): Promise<void> => {
 	log.info('event', { event })
-	await repo.extendTTL(event.requestContext.connectionId)
+	try {
+		await repo.extendTTL(event.requestContext.connectionId)
+	} catch (error) {
+		if (error instanceof ConditionalCheckFailedException) {
+			log.debug(`WebConnection is not found`, {
+				context: event.requestContext.authorizer,
+			})
+			track('ConnectionIdMissing', MetricUnits.Count, 1)
```

Comment:
*problem:* it is not checked why `extendTTL` fails. It may not be cause by a missing connectionId.

Reply:
This metric will be logged only when the error is `ConditionalCheckFailedException` as per https://github.com/hello-nrfcloud/backend/pull/234/files/c1bfc89e39a270e86f7ce708d301f85b62453a03#diff-900a11ad083b712ce3f9a5b3e1df19c6723e1edae68a0be3d0d9a38252cf2ec4R104.

If error is something else, it will just ignore.